docs/docs/sourcing-from-kentico-kontent.md

```diff
@@ -0,0 +1,228 @@
+
+You can also take advantage of Kentico Kontent's ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kentico Kontent, the [source plugin](https://github.com/Kentico/gatsby-source-kontent) will ensure the proper nodes are created for your Gatsby site.
+
+_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kentico Kontent works. If you'd like to look at an example of a complete basic site, have a look at the [Kentico Cloud Gatsby starter site](https://github.com/Kentico/gatsby-starter-kontent), which includes worked examples for querying content._
+
+## Setup
+
+### Kentico Kontent
+
+The first thing to do, if you haven't already done so, is [sign up for a Kentico Kontent account](https://app.kontent.ai/sign-up). This will automatically start a free 30-day trial with all Kentico Kontent features. At any point during the trial or after, you can switch to a Starter plan (which always starts as free) or a higher plan with more features.
+
+Once you have a subscription to Kentico Kontent, you need some content to retrieve. If you know what you want, you can set up your own content types (templates for your content) and then create content items (the actual content) based on them. If you'd like to take a shortcut and see some example content, you can use our [Sample Project generator](https://app.kontent.ai/sample-project-generator) to create sample content. This guide will continue with the example of the Sample Project.
+
+The created Sample Project is a comprehensive presentation of a fictional coffee company named Dancing Goat that showcases various Kentico Kontent features. It can be displayed in various channels, as you can see by going to the [Quickstart page in Kentico Kontent](https://app.kontent.ai/quickstart) from within that project.
+
+For this guide, you don't have to worry about most of the features. You'll just pull some data to display within the Gatsby site you'll create in the next step. The only thing you need to continue is your Project ID, which you can find in Kentico Kontent under _Project settings_ -> _API keys_.
+
+### Gatsby
+
+#### Adding content to existing pages
+
+Now that you have some content to pull, you can create a basic Gatsby site to display the content. Assuming you have the [Gatsby CLI installed](/docs/quick-start/#install-the-gatsby-cli), create a new site and navigate to it in your terminal:
+
+```shell
+gatsby new kentico-kontent-guide
+cd kentico-kontent-guide
+```
+
+Next, install the [Kentico Kontent source plugin](https://github.com/Kentico/gatsby-source-kontent):
+
+```shell
+yarn add @kentico/gatsby-source-kontent
```

Comment:
This was changed from `npm install` in previous version, we prefer using `npm` over `yarn` in our documentation whenever `yarn` is not actually needed (for cases like yarn workspaces), as `npm` is the default package manager (and likely developers using `yarn` will know how to translate `npm` command to `yarn` if needed).

Reply:
Sorry about that.